corehq/apps/toggle_ui/views.py

```diff
@@ -194,6 +197,26 @@ def post(self, request, *args, **kwargs):
         toggle.save()
 
         changed_entries = previously_enabled ^ currently_enabled  # ^ means XOR
+
+        added_entries = currently_enabled - previously_enabled
+        is_deprecated_toggle = (self.static_toggle.tag == TAG_DEPRECATED)
```

Comment:
I could imagine this also applying to TAG_CUSTOM and TAG_INTERNAL - anecdotally, I feel like a number of the issues we have are when a flag built for a project like ICDS starts getting used on solutions projects. Or would that lead to too many emails?